TestFoundation/TestCodable.swift

```diff
@@ -318,7 +319,12 @@ class TestCodable : XCTestCase {
             // causing encode -> decode -> compare test to fail.
             values.append(TimeZone.current)
         #endif
-        
+#else
+        var values = [
+            TimeZone(identifier: "UTC")!,
+            TimeZone.current
+            ]
```

Comment:
I guess the problem here is that Android doesn't come with a reasonable ICU?

Reply:
Indeed, options are very limited for Android date/time conversion from C